### ISSUE TITLE
Building node on Windows: Cannot read property 'PYTHON' of undefined

### ISSUE THREAD

**Issue by s-h-a-d-o-w** [nexe/nexe#493]
This is a
 - [x] Bug Report
 - [ ] Feature Request
 - [ ] Other

<br/>
If this is a bug report, What are the steps to reproduce?

Simply building:
```
D:\development\node playground\dummy>nexe bla.js -b -f="--expose-gc"
i nexe 2.0.0-rc.28
\ Downloading Node.js source from: https://nodejs.org/dist/v8.11.2/node-v8.11.2.
| Downloading Node.js source from: https://nodejs.org/dist/v8.11.2/node-v8.11.2.
/ Downloading Node.js source from: https://nodejs.org/dist/v8.11.2/node-v8.11.2.
- Downloading Node.js source from: https://nodejs.org/dist/v8.11.2/node-v8.11.2.
\ Downloading Node.js source from: https://nodejs.org/dist/v8.11.2/node-v8.11.2.
V Node source extracted to: C:\Users\Brix\.nexe\8.11.2
- Compiling result
TypeError: Cannot read property 'PYTHON' of undefined
    at NexeCompiler._configureAsync (D:\development\github\nexe\lib\compiler.js:
213:52)
    at NexeCompiler.<anonymous> (D:\development\github\nexe\lib\compiler.js:224:
51)
    at step (D:\development\github\nexe\lib\compiler.js:46:23)
    at Object.next (D:\development\github\nexe\lib\compiler.js:27:53)
    at D:\development\github\nexe\lib\compiler.js:21:71
    at new Promise (<anonymous>)
    at __awaiter (D:\development\github\nexe\lib\compiler.js:17:12)
    at NexeCompiler._buildAsync (D:\development\github\nexe\lib\compiler.js:218:
16)
    at NexeCompiler.<anonymous> (D:\development\github\nexe\lib\compiler.js:311:
51)
    at step (D:\development\github\nexe\lib\compiler.js:46:23)
```

Looks like if you don't explicitly set the `python` option, this line is never executed:
`this.env = { ...process.env }`
... causing the error above when using `build`.

Please also provide:

- Platform(OS/Version): Win7 x64
- Host Node Version: 8.11.2
- Target Node Version: 8.11.2
- Nexe version: 2.0.0-rc.28
- Python Version: 2.7.11

**Comment by calebboyd** [comment]
A fix has been published. You can update to fix this

**Comment by wbobeirne** [comment]
Are you sure this is fixed? I just ran into this with `2.0.0-rc.27` and it looks like this conditional is still here, if you don't specify `python` then it'll never trigger, which means `this.env` is never set: https://github.com/nexe/nexe/blob/v2.0.0-rc.27/src/compiler.ts#L59

**Comment by calebboyd** [comment]
Sorry, Forgot to drop git tags for later versions.
https://github.com/nexe/nexe/commit/089aa02a8208907db0a4145715896a72383197e1

Go ahead and update.
